### PR TITLE
Feature/SM-1 Implemented the login story

### PR DESCRIPTION
**Pull request by paulcaraa**

### CODE REVIEW

**Comment by cubeti** [comment]
Resolve the conflics

File: src/loose/oose/fis/lab/student/manager/Controller.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package loose.oose.fis.lab.student.manager;
```
Comment (cubeti) [inline src/loose/oose/fis/lab/student/manager/Controller.java]: remove unused file

File: src/loose/oose/fis/lab/student/manager/sample.fxml [hunk 2]
```diff
@@ -3,6 +3,6 @@
```
Comment (cubeti) [inline src/loose/oose/fis/lab/student/manager/sample.fxml]: remove unused file

File: src/loose/oose/fis/lab/student/manager/controllers/LoginController.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+            loginMessage.setText("Logged in as student!");
+            return;
+        }
+        if (username.equals("teacher") && password.equals("teacher")) {
+            loginMessage.setText("Logged in as teacher!");
+            return;
+        }
+        loginMessage.setText("Incorrect login!");
+    }
+}
```
Comment (cubeti) [inline src/loose/oose/fis/lab/student/manager/controllers/LoginController.java:52]: If login as a teacher succeeds, the user should be taken to the view-students scene

**Review by cubeti — CHANGES_REQUESTED** [review]
Fix the comments

**Comment by paulcaraa** [comment]
Solved!

**Review by cubeti — APPROVED** [review]
Gj

**Comment by cubeti** [comment]
ok approved

**Pull request merged**